app-backend/datamodel/src/main/scala/com/azavea/rf/datamodel/ColorCorrect.scala

```diff
@@ -18,6 +24,20 @@ object ColorCorrect {
       (tile.subsetBands(redBand, greenBand, blueBand), Array(hist(redBand), hist(greenBand), hist(blueBand)))
   }
 
+  object Params {
+    implicit val defaultColorCorrectParamsFormat = jsonFormat13(Params.apply _)
+
+    def colorCorrectParams: Directive1[Params] =
```

Comment:
@notthatbreezy These are used from both app and tiler now. Seems a little weird to have them in the datamodel, but no other less weird place can be found. Looking for :+1: to this as a thing.